### DISCUSSION TITLE
Send messages?

### DISCUSSION THREAD

**Discussion by alexanderradahl** [subzeroid/aiograpi-rest#27] [Q&A] [answered]
Hi there! Is there anyway to send messages? :)

**Comment by adw0rd** [comment, marked as answer]
Not implemented in instagrapi-rest, you can try https://lamadava.com

![image](https://user-images.githubusercontent.com/546889/143788929-a108705d-1eac-4776-b668-7b98f0bc5271.png)

**Reply by alexanderradahl** [reply]
Is there anyway to post comment?

**Reply by adw0rd** [reply]
added following endpoints:

![image](https://user-images.githubusercontent.com/546889/143949282-33cd7233-5535-4ae6-8c8c-254b8f4dcbcd.png)

but for public comments I suggest use `/v1/media/comments`:

![image](https://user-images.githubusercontent.com/546889/143949576-4dfcd9a5-1ddf-422a-a0e2-c27333d875fb.png)

**Reply by mikebgrep** [reply]
@adw0rd Is still direct message not implemented in instagrapi-rest

**Reply by adw0rd** [reply]
@mikebgrep you can help for this project

**Reply by mikebgrep** [reply]
I am Java at the moment Python is like a hobby otherwise not a problem

**Reply by mikebgrep** [reply]
@adw0rd  after once deployed if the Instagram block the device is have a way to change it.

**Reply by adw0rd** [reply]
@mikebgrep it's question? You can change settings (including device) https://github.com/adw0rd/instagrapi-rest/blob/main/routers/auth.py#L53

**Reply by mikebgrep** [reply]
That make sense. Thank you
